### ISSUE TITLE
smos-archive Ignores Config's workflow-dir?

### ISSUE THREAD

**Issue by prikhi** [NorfairKing/smos#54]
I was testing some aliases/functions I wrote and I can't seem to get `smos-archive` to work with my custom workflow dir `~/.tasks`:

```
$ cd ~
$ smos-archive .tasks/test.smos
The smos file
/home/prikhi/.tasks/test.smos
is not in the smos workflow directory
/home/prikhi/workflow/

$ grep 'workflow-dir' .config/smos/config.yaml --context=1
workflow-dir:
    "/home/prikhi/.tasks"

$ smos-archive .tasks/test.smos --config-file .config/smos/config.yaml
The smos file
/home/prikhi/.tasks/test.smos
is not in the smos workflow directory
/home/prikhi/workflow/

$ smos-archive .tasks/test.smos --workflow-dir ~/.tasks/
The smos file
/home/prikhi/.tasks/test.smos
is not in the smos workflow directory
/home/prikhi/workflow/
```

Am I doing or setting something incorrectly or is this a bug in `smos-archive`?

**Comment by NorfairKing** [comment]
It looks like you're right. I made this regression test: a9d91297a31d7c7ab546ef6016f8cb34a5c257a5

**Comment by NorfairKing** [comment]
This should be fixed after 9053fbfcc. Please check and then close this issue.

**Comment by prikhi** [comment]
Yup, works for me - thanks!
